release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -467,7 +467,32 @@ In the table, features are marked with the following statuses:
 === Removed features
 
 [id="ocp-4-6-bug-fixes"]
-== Bug fixes
+== Bug fixes 
+
+*Web console (Developer perspective)*
+
+* Previously, when you tried to delete a Knative application through a *Topology* view, a false positive error about an non-existing *Knative route* was reported. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1866214[BZ#1866214])
```

Comment:
through a *Topology* view/through the *Topology* view
an non/a non.
Please add : This issue is now fixed and the error is no longer displayed. or something to that effect.

Reply:
```suggestion
* Previously, when you tried to delete a Knative application through the *Topology* view, a false positive error about a non-existing *Knative route* was reported. This issue is now fixed and the error is no longer displayed. link:https://bugzilla.redhat.com/show_bug.cgi?id=1866214[BZ#1866214])
```